### PR TITLE
qt imports fix

### PR DESCRIPTION
**Pull request by JGoutin**
Related to this pull request : 
https://github.com/matplotlib/matplotlib/pull/4912

### CODE REVIEW

**Comment by tacaswell** [comment]
@jrevans Does this fix your problems?

File: lib/matplotlib/backends/qt_compat.py [hunk 1]
```diff
@@ -26,8 +27,18 @@
...
 if rcParams['backend'] == 'Qt5Agg':
     QT_RC_MAJOR_VERSION = 5
-else:
+elif rcParams['backend'] == 'Qt4Agg':
     QT_RC_MAJOR_VERSION = 4
+else:
+    # A different backend was specified, but we still got here because a Qt
+    # related file was imported.  This is allowed, so lets try and guess
+    # what we should be using.
+    if "PyQt5" in sys.modules:
```
Comment (tacaswell) [inline lib/matplotlib/backends/qt_compat.py]: Can you invert this logic to check for pyqt4 and default to 5?

Comment (JGoutin) [inline lib/matplotlib/backends/qt_compat.py]: OK, now pyQt5 is default.

File: lib/matplotlib/backends/qt_compat.py [hunk 2]
```diff
@@ -26,8 +27,18 @@
...
 if rcParams['backend'] == 'Qt5Agg':
     QT_RC_MAJOR_VERSION = 5
-else:
+elif rcParams['backend'] == 'Qt4Agg':
     QT_RC_MAJOR_VERSION = 4
+else:
+    # A different backend was specified, but we still got here because a Qt
+    # related file was imported. This is allowed, so lets try and guess
+    # what we should be using.
+    if "PyQt4" in sys.modules:
```
Comment (tacaswell) [inline lib/matplotlib/backends/qt_compat.py]: This also needs to check for `PySide`, sorry I did not realize that before.

Comment (JGoutin) [inline lib/matplotlib/backends/qt_compat.py]: For the PyQt4 lines ?
I never used PySide. So, don't know its specificities.

Is the following change correct ?
```python 
if "PyQt4" in sys.modules:
```
=>
```python
if "PyQt4" in sys.modules or "PySide" in sys.modules:
```

Comment (tacaswell) [inline lib/matplotlib/backends/qt_compat.py]: Yes, that looks like.

Comment (JGoutin) [inline lib/matplotlib/backends/qt_compat.py]: PySide added.

**Comment by tacaswell** [comment]
Thanks!

**Pull request merged**